### DISCUSSION TITLE
Compatibility matrix for Image Updater and ArgoCD

### DISCUSSION THREAD

**Discussion by radimpaseka** [argoproj-labs/argocd-image-updater#868] [Q&A]
Is there a compatibility matrix for Image Updater and ArgoCD, can I find out if my version of Image Updater will not work with some version of ArgoCD?

**Comment by jannfis** [comment]
I'm sorry, we don't have any compatibility matrix right now.

As we have limited resources, we strive to support the current stable version of Argo CD with each minor release.

But your point is valid. I think by the time of a GA release, we should figure out an official compatibility/support strategy.
